src/components/Composer/index.js

```diff
@@ -119,22 +109,20 @@ class Composer extends React.Component {
     constructor(props) {
         super(props);
 
-        const initialValue = props.defaultValue
-            ? `${props.defaultValue}`
-            : `${props.value || ''}`;
+        this.selection = {
+            start: props.defaultValue.length,
+            end: props.defaultValue.length,
+        };
```

Comment:
`this.selection` is assigned here, but never update or used in the component 

I think the style guide says to remove unused stuff

Reply:
Oh good catch, thx!